01/credit/credit.c

```diff
@@ -0,0 +1,61 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdlib.h>
+
+int main(void){
+
+    long long int cardNumberInt;
+    printf("Number: ");
+    scanf("%lld", &cardNumberInt);
+
+    long long int length = floor(log10(llabs(cardNumberInt))) + 1;
```

Comment:
Die Länge braucht keinen long long int ist ja nicht größer als zwei milliarden
```suggestion
    int length = floor(log10(llabs(cardNumberInt))) + 1;
```